src/lib/crypto/kimchi_backend/gadgets/foreign_field.mli

```diff
@@ -19,6 +19,26 @@ module Snark_intf = Snarky_backendless.Snark_intf
  *)
 type 'field standard_limbs = 'field * 'field * 'field
 
+type 'field compact_limbs = 'field * 'field
+
+val bignum_bigint_to_field_const_standard_limbs :
```

Comment:
Matthew suggested that the names should be more clear about what is constant.  The idea is to rename them now so that it's recorded in the code so that later when we refactor into `hlist` and `Blah.typ` and `Blah.typ_unchecked` we will automatically know what's const and what's not. 👍🏼